pkg/virt-handler/options.go

```diff
@@ -47,6 +47,41 @@ func virtualMachineOptions(
 	return options
 }
 
+func virtualMachineOptionsWithNetworkConfig(
```

Comment:
I think it was raised before, couldn't find the thread.
It will be nicer to convert this function to an option function of `virtualMachineOptions()` (i.e.: turn virtualMachineOption() to builder function).